lib/cocoapods/installer/installation_options.rb

```diff
@@ -153,6 +153,11 @@ def hash
       #
       option :warn_for_multiple_pod_sources, true
 
+      # Whether to emit a warning if a project is not explicitly specifying the git based master specs repo and can
+      # instead use CDN which is the default.
+      #
+      option :warn_for_unused_master_specs_repo, true
```

Comment:
I added this option because its possible that users might have multiple projects. One that uses master specs repo and one that doesn't. This will allow them not to have warnings and not ask them to delete the master specs repo.